### PR TITLE
Grammar and spelling fixes

### PR DESCRIPTION
**Pull request by rm-rf-home**
"Kündigen" is a better translation for "unsubscribe" because "Abmelden" usually means "log out".
No need to use formal "Sie", "Du" is now used for all translations

- [X] I have read and agree to the [Contribution Guidelines](https://github.com/FreeTubeApp/FreeTube/blob/master/CONTRIBUTING.md)
- [X] I can maintain / support my code if issues arise.

**Notes and Comments about your pull request**

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
 	"View": "Wiedergabe",
 	"Views": "Wiedergaben",
 	"Subscribe": "Abonnieren",
-	"Unsubscribe": "Abmelden",
+	"Unsubscribe": "Kündigen",
```
Comment (TobiGr) [inline locales/de.json]: I'd say "Deabonieren". This is not contract or something similar

Comment (rm-rf-home) [reply locales/de.json]: True, "kündigen" sounds more like cancelling a contract. Then I would suggest to use "Abo beenden". This is also what is used on YouTube.

```suggestion
	"Unsubscribe": "Abo beenden",
```

Comment (TobiGr) [reply locales/de.json]: Sounds good 👍

**Comment by PrestonN** [comment]
Great!  Thanks you!

**Pull request merged**